Library/Formula/tccutil.rb

```diff
@@ -0,0 +1,17 @@
+class Tccutil < Formula
+  desc "Utility to modify the OS X Accessibility Database (TCC.db)"
+  homepage "https://github.com/jacobsalmela/tccutil"
+  url "https://github.com/jacobsalmela/tccutil/archive/v1.2.0.tar.gz"
+  sha256 "3fcac0c458969629366002a2e0142a7291ab44a0077c5495de4b1383704861bc"
+  head "https://github.com/jacobsalmela/tccutil.git"
+
+  bottle :unneeded
+
+  def install
+    bin.install "tccutil.py" => "tccutil"
```

Comment:
I should add that technically it is not replacing but rather extending the built-in utility. The built-in utility only supports one command: `reset`.

This command is passed through by the utility.

If this doesn't change anything, I'd still rather add it to `homebrew/dupes` as I think it's clearer that way.